interp/runt.toml

```diff
@@ -83,6 +72,47 @@ cmd = """
 """
 expect_dir = "tests/lowered/"
 
+
+
+[[tests]]
+name = "unit (CIDB)"
+paths = [
+  "tests/unit/*.futil"
+]
+cmd = """
+  ../target/debug/interp {} debug -p | jq .memories
```

Comment:
Just double-checking: these test setups somehow make the debugger just step until the programs are done, right? So they kinda force the debugger into a configuration where it works like the normal interpreter, and the tests ensure that it indeed produces the same output as the normal interpreter—right?